pom.xml

```diff
@@ -648,6 +649,9 @@
 							<goals>
 								<goal>jar</goal>
 							</goals>
+							<configuration>
```

Comment:
As the additional option is only required for Java 8 (and probably Java >= 8), I would have added a pluginManagement section to a `jdk_greater_than_eight` profile. It seems to me more readable than having a property that is emptied for specific cases (too many indirection and negation for my taste).

If you feel that your way is the best, I'll merge as is...